src/reanimated2/commonTypes.ts

```diff
@@ -36,12 +46,12 @@ export type ShareableSyncDataHolderRef<T> = {
   __hostObjectShareableJSRefSyncDataHolder: T;
 };
 
-export type MapperRegistry = {
+export type MapperRegistry<T> = {
   start: (
     mapperID: number,
     worklet: () => void,
-    inputs: SharedValue<any>[],
-    outputs?: SharedValue<any>[]
+    inputs: Record<string, unknown>[],
```

Comment:
is it really correct?

Reply:
Apparently, yes!

![Screenshot 2023-12-08 at 10 41 17](https://github.com/software-mansion/react-native-reanimated/assets/40713406/5bfec1af-aad7-492e-a6ed-f3a88029dc8b)